src/main/cpp/Constants.cpp

```diff
@@ -0,0 +1,42 @@
+#include "Constants.h"
+
+namespace RobotConstants {
+    // although I think getPosition() might return in native units of
+    // rotations so this would then be 1
+    const double neo550TicksPerRev = 42;
```

Comment:
you are correct
https://codedocs.revrobotics.com/cpp/classrev_1_1_c_a_n_encoder.html#a02c84657c4c771902f8af262bbf2be7e

Reply:
This comment is copypasta, so you are correct that you are correct